app/src/main/java/com/malmstein/yahnac/stories/ArticleActivity.java

```diff
@@ -35,6 +35,8 @@
     private int croutonAnimationDuration;
     private int croutonBackgroundAlpha;
 
+    private Menu menu;
```

Comment:
This field is not needed. Let's play by the framework's rules instead (see [comment below](https://github.com/malmstein/yahnac/pull/13/files#r27360643)).